api/routes/batch/types.go

```diff
@@ -21,3 +21,10 @@ type Event struct {
 	BatchID    string `json:"batch_id"`
 	ResourceID string `json:"resource_id"`
 }
+
+// Response describes a batch in request response payload
+type Response struct {
+	Id        string   `json:"id"`
+	Timestamp string   `json:"timestamp,omitempty"`
+	Resources []string `json:"resources,omitempty"`
+}
```

Comment:
I find it weird to use the same type for responses that are different (timestamp is only used in the "list" route, and resources only in the "resources" route). The type is not really meaningful here, what do you think ?

Reply:
I don't know, I thought It would be nice to have a common structure. It may be useful if we plan to extend the batch api.